tasks/section4.yml

```diff
@@ -468,8 +499,10 @@
       - rule_4.2.4
 
 - name: "NOTSCORED | 4.3 | PATCH | Ensure logrotate is configured"
-  command: /bin/true
-  changed_when: no
+  lineinfile:
+    path: /etc/logrotate.conf
+    regexp: '^(daily|weekly|monthly|yearly)$'
+    line: 'daily'
```

Comment:
I think this rule needs to take into account both `/etc/logrotate.conf` and items in `/etc/logrotate.d/`. Also per rule 4.3 it doesn't specify that logs need to be rotated daily just that they should be configured to rotate `per site policy`.  Maybe make the the rotation interval configurable?

Reply:
Good point, I have updated now.

Reply:
For some reason this is failing on Travis, but works in my vagrant and on a clean vm. Any ideas?